packages/atomic/stencil.config.ts

```diff
@@ -17,6 +34,7 @@ const isDevWatch: boolean =
 export const config: Config = {
   namespace: 'atomic',
   taskQueue: 'async',
+  globalScript: 'src/global/environment.ts',
```

Comment:
Stencil will do a default import on the `globalScript` file and call it.

Source: [Stencil documentation](https://stenciljs.com/docs/config#globalscript)